README.md

```diff
@@ -90,6 +92,20 @@ const checkoutInstance = startCheckout(manifest, {
 })
 ```
 
+### On Close
+
+```js
+function myCloseHandler(checkoutCloseMessage) {
+  if (checkoutCloseMessage === 'close') {
```

Comment:
We ended up removing the parameter (checkoutCloseMessage) so it can be deleted here as well.

Reply:
done